### ISSUE TITLE
No flow match "Login with existing keycard" in status desktop

### ISSUE THREAD

**Issue by churik** [status-im/status-react#21756]
# Bug Report

## Problem 
Currently for a flows that not implemented on status mobile there is a pop up that we need to go to status-desktop and check the flow there.

But currently the only matching flow is " Add new user > I don't have other device > login with keycard" and it has some issues. 

#### Expected behavior
can use exisiting keycard to login to desktop

#### Actual behavior
https://github.com/user-attachments/assets/45345887-cb4f-4643-b624-2ba9796b2f62

### Reproduction

1. Desktop: "Add new user > I don't have other device > login with keycard"  
2. Try to use keycard
3. It stuck, the only way I can use it is to reopen Status app
4. in this case user will be there, but with 3 random name and only with default account in wallet. 

### Additional Information

- Status version: master 04/12
- Operating System: Mac OSc

Logs from desktop:
[console.log](https://github.com/user-attachments/files/18011022/console.log)
[geth copy 3.log](https://github.com/user-attachments/files/18011053/geth.copy.3.log)

**Comment by guylouis** [comment]
so the problem here is twofold:  
- the behaviour doesn't work straight
- the copy is very hard to understand for someone who has an existing keycard and just wants to login with his keycard

right ?

**Comment by churik** [comment]
@guylouis yes, it can work or not, depending if the datais stored in waku + it is very unclear that in order to login with keycard to desktop you need to pick "I do not have other device" (which they do) > Login with keycard

**Comment by anastasiyaig** [comment]
the process is not stuck, it is trying to fetch profile data from waku. the duration of that is 2 min. since there is no metadata for account from mobile, fetching is not returning anything and asks you to create new profile with the same chat key.

**Comment by churik** [comment]
@guylouis given @anastasiyaig input we can somehow direct users use this flow with clear instructions (for me it was not clear), that

1) data might be not restored and in this case you need to pick the flow "Create a new profile with same chatkey"
2) in this case no data will be recovered from waku and your additional keypairs /accounts will be not recovered (but they can try to add it manually)
3) in this case the profiles between desktop <> mobile will be not paired, so the recommendation would be not to use this flow if the users plan to use any other features in the app (chatting, communites, etc)

I'm just not sure how to make it clear for users, my assumption that for now it is just better to avoid this flow until implementing sync on mobile and resolving discussion about metadata.

**Comment by anastasiyaig** [comment]
i would agree the instructions on that screen are confusing. These screens are quite old to be honest. Any quick improvement to bring clarity for users wont hurt for sure, maybe @benjthayer could help with any robust solutions?

**Comment by anastasiyaig** [comment]
@churik lets separate things a bit here:

case1: user has keypairs , imported seed to status desktop.
In this case we will restore keypairs (they would be found in settings-wallet-keypairs, you will be prompted to reimport the seeds for non status keypairs)

case2: mobile keypair migrated
to keycard and then user logs in with that keycard. In this case you cant expect multiple keypairs restored, as you migrated to keycard only one. We cant still get metadata from mobile (there is none), but we indicate that the wallet account that belongs to that keypair, is from keycard (icon in wallet screen)

general solution how to handle missing metadata is separate task , we will address that separately

**Comment by anastasiyaig** [comment]
@churik check here https://github.com/status-im/status-desktop/pull/16908 some more
explanation on that matter (regarding keypairs count)

**Comment by anastasiyaig** [comment]
ah also one more thing, that we store data in waku for 30 days. So if user did not come online for 30 days in a row, nothing will be restored from waku and thats expected. However, there is no single place in the app user could learn about.

i apologize for spam to that ticket 🙏 , texting from my phone now

**Comment by benjthayer** [comment]
Hey @anastasiyaig @jrainville in collaboration with @guylouis , I've proposed these small copy changes to the sign in flows to make it a little more intuitive where the user will find the option to sign in via Keycard. We could refine this a lot more but these screens will be replaced by the new onboarding in 2.33 which inherently make it a lot easier to find this option.

Summary of changes:
- Replace title `Sign in by syncing` with `Log in by syncing`
- Replace CTA label `I don't have other device` with `Log in via Keycard or seed phrase instead`
- Replace title on screen 3 `Connect your keys` with `Log in`
- Replace subtitle on screen 3 `Use your existing Status keys to login to this device` with `Use your existing Status keys to log in to this device`
![Image](https://github.com/user-attachments/assets/d8fccb90-4d5c-4dbf-884c-1849136f0e91)

https://www.figma.com/design/17fc13UBFvInrLgNUKJJg5/Kuba%E2%8E%9CDesktop?node-id=22455-509532&t=qzcAVBYDPKOBCd6M-4

**Comment by churik** [comment]
oh that can be closed I believe, it was done on desktop
